_docs/quickstarts/scanner/descriptions.md

```diff
@@ -0,0 +1,131 @@
+[`firewall_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/firewall_rules.yaml)
+rule file.
+
+## Load Balancer Forwarding Rules scanner
+
+Load balancer forwarding rules can be configured to direct unauthorized external
+traffic to your target instances. The forwarding rule scanner supports a
+whitelist mode, to ensure each forwarding rule only directs to the intended
+target instances.
+
+For examples of how to define scanner rules for your forwarding rules, see the
+[`forwarding_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/forwarding_rules.yaml)
+rule file.
+
+## Groups scanner
+
+Because groups can be added to Cloud Identity and Access Management (Cloud IAM)
+policies, G Suite group membership can allow access on Google Cloud Platform (GCP).
+The group scanner supports a whitelist mode, to make sure that only authorized
+users are members of your G Suite group.
+
+For examples of how to define scanner rules for your G Suite groups, see the
+[`group_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/group_rules.yaml)
+rule file.
+
+## IAM policy scanner (organization resources)
+
+Cloud IAM policies directly grant access on GCP. To ensure only authorized
+members and permissions are granted in Cloud IAM policies, IAM policy scanner
+supports the following:
+
+ - Whitelist, blacklist, and required modes.
+ - Define if the scope of the rule inherits from parents or just self.
+ - Access to specific organization, folder, or project resource types.
+
+For examples of how to define scanner rules for Cloud IAM policies, see the
+[`iam_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/iam_rules.yaml)
+rule file.
+
+## IAP scanner
+
+Cloud Identity-Aware Proxy (Cloud IAP) enforces access control at the network
+edge, inside the load balancer. If traffic can get directly to the VM, Cloud IAP
+is unable to enforce its access control. The IAP scanner ensures that firewall
+rules are properly configured and prevents the introduction of other network
+paths that bypass the normal load balancer to instance flow.
+
+For examples of how to define scanner rules for Cloud IAP, see the
+[`iap_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/iap_rules.yaml)
+rule file.
+
+## Instance Network Interface scanner
+
+VM instances with external IP addresses expose your environment to an
+additional attack surface area. The instance network interface scanner audits
+all your VM instances in your environment, and determines if any VMs with
+external IP addresses are outside of the trusted networks.
+
+For examples of how to define scanner rules for network interfaces, see the 
+[`instance_network_interface_rules.yaml`](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/rules/instance_network_interface_rules.yaml)
+rule file.
+
+## KE Version scanner
```

Comment:
Is this an approved or official name for the scanner? We aren't supposed to abbreviate "Kubernetes Engine" per branding specifications.

Reply:
Updated.